src/components/QuillForms/RegisterClientForm.tsx

```diff
@@ -574,12 +574,15 @@ export default function RegisterClientForm({ source }: { source: string | string
                         ...(contactPreferenceAnswer?.includes('call') || contactPreferenceAnswer?.includes('whatsapp')
                             ? [
                                   {
-                                      name: 'number',
+                                      name: 'short-text',
```

Comment:
im making it a text so that we can add the +351 or whatever if we need.

And also this helps with the minLength part of it. Numbers don't have a length